bootc/podman_helper.sh

```diff
@@ -115,14 +125,18 @@ Time for video
 }
 
 function create_disk_image {
+    storedir=$(pwd)/store
+    sudo mkdir -p ${storedir}
```

Comment:
my mac did not like set to `/tmp/store` i don't know why 🤷

Reply:
I think you want the storedir to be on the local host system on the mac.